### ISSUE TITLE
Plans to support symfony/mailer on this package

### ISSUE THREAD

**Issue by abishekrsrikaanth** [jdavidbakr/mail-tracker#155]
@jdavidbakr 

With Laravel 9 approaching soon, the `swiftmailer/swiftmailer` package is being removed and `symfony/mailer` will be used. 
Any plans to get this package ready with the release on the new Laravel version?

**Comment by jdavidbakr** [comment]
Going to have to wait on the 9.x-dev branch of Laravel, and orchestra/testbench to support Laravel 9.

**Comment by jdavidbakr** [comment]
I've been swamped and haven't had a chance to investigate what needs to happen to change to use symfony/mailer instead of swiftmailer/swiftmailer.  If someone wants to investigate and submit a pull request I will be happy to accept it, otherwise I will get this updated once I have some time to do so.

**Comment by jdavidbakr** [comment]
Folks, I'm going to need some help from someone who knows a bit more about the Symfony Mailer.  After digging for quite a while I don't see a way to hook in to modify the outgoing email the way you could with the SwiftMailer.  Anyone want to at least pass some knowledge about how I can keep this package alive?

**Comment by jdavidbakr** [comment]
Thanks - I will look at this as a possible solution!

**Comment by jdavidbakr** [comment]
I have the code updated, dependent on https://github.com/laravel/framework/pull/41615 being merged into 9.x

**Comment by jdavidbakr** [comment]
Released: https://github.com/jdavidbakr/mail-tracker/releases/tag/6.0
